cclib/bridge/cclib2pyquante.py

```diff
@@ -1,31 +1,72 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2017, the cclib development team
+# Copyright (c) 2020, the cclib development team
 #
 # This file is part of cclib (http://cclib.github.io) and is distributed under
 # the terms of the BSD 3-Clause License.
 
 """Bridge for using cclib data in PyQuante (http://pyquante.sourceforge.net)."""
 
+import numpy
 from cclib.parser.utils import find_package
 
+
+class MissingAttributeError(Exception):
+    pass

```

Comment:
Should this go elsewhere like `cclib/parser/data.py` or something so we can use this in other methods in the future? @langner @berquist

Reply:
Just an fyi, I've copied this from `cclib.method.calculationmethod` -- most methods use this as well.